Week7/week7-solution.txt

```diff
@@ -0,0 +1,89 @@
+#1. Add 12 Records. Take input from user.
+import csv
+import pandas as pd
+
+header = ["Prod_No", "Prod_Name", "Jan", "Feb", "Mar", "Apr", "May", "Jun"]
+data=[[1,'PENCIL',100,200,200,350,200,300],[2,'PEN',200,400,400,350,200,300],[3,'TV',10,10,20,35,20,10],[4,'SOAP',100,200,200,350,200,300],[5,'SHAMPOO',1000,2000,200,350,2000,3000]]
+
+with open('c:\sqlite3\csv\week7.csv','w',newline='') as f:
+    w=csv.writer(f)
+    w.writerow(header)
+    w.writerows(data)
+    for i in range(12):
+        no=int(input("Enter Product No. :"))
+        name=input("Enter Product Name :")
+        jan=int(input("Enter January sell :"))
+        feb=int(input("Enter February sell :"))
+        mar=int(input("Enter March sell :"))
+        apr=int(input("Enter April sell :"))
+        may=int(input("Enter May sell :"))
+        june=int(input("Enter June sell :"))
+        sub_lis=[no,name,jan,feb,mar,apr,may,june]
+        w.writerow(sub_lis)
+
+with open('c:\sqlite3\csv\week7.csv','r',errors='ignore') as f:

```

Comment:
what is  the use of errors='ignore'....?

Reply:
it is used to ignore the error and continue the program process